mteb/evaluation/evaluators/model_classes.py

```diff
@@ -338,17 +320,20 @@ def _full_corpus_search(
             logging.info("Computing Similarities...")
             query_embeddings = torch.as_tensor(query_embeddings).to(device)
             sub_corpus_embeddings = torch.as_tensor(sub_corpus_embeddings).to(device)
+
+            score_function = (
+                self.model.score if hasattr(self.model, "score") else cos_sim
```

Comment:
This was the line that confused me, but I am only on mobile today so perhaps I’m misreading

Reply:
You are right. This should be `similarity` function as in sentence_transformers